regexes/client/libraries.yml

```diff
@@ -566,3 +566,18 @@
   name: 'request'
   version: ''
   url: 'https://github.com/request/request'
+
+- regex: 'kiwi-tcms/([a-z\d\.A-Z]+)'
+  name: 'Kiwi TCMS'
+  version: '$1'
+  url: 'https://kiwitcms.org'
+
+- regex: 'tcms-api/([a-z\d\.A-Z]+)/Python(.*)'
+  name: 'Kiwi TCMS API clients'
+  version: '$1'
+  url: 'https://kiwitcms.org'
+
+- regex: 'kiwitcms\/gitops/([a-z\d\.A-Z]+)'
+  name: 'Kiwi TCMS GitOps bot'
+  version: '$1'
+  url: 'https://kiwitcms.org'
```

Comment:
```suggestion
- regex: 'kiwitcms/gitops/[\d.]+'
  name: 'Kiwi TCMS GitOps'
  category: 'Service Agent'
  url: 'https://kiwitcms.org/'
  producer:
    name: 'Open Technologies Bulgaria, Ltd'
    url: 'https://kiwitcms.org/'
```

Reply:
You have to move this regex to `regexes/bots.yml` and add a test user-agent to `Tests/fixtures/bots.yml`.